packages/scikit-image/test_skimage.py

```diff
@@ -50,3 +50,4 @@ def test_skimage():
     assert len(np.unique(segments_fz)) == 194
     assert len(np.unique(segments_slic)) == 196
     assert len(np.unique(segments_quick)) == 695
+    assert len(np.unique(segments_watershed)) > 0  # TODO: some number, perhaps?
```

Comment:
I think we should figure out what `len(np.unique(segments_watershed))` is and put that number here most likely.

Reply:
These run in unittest, so these asserts don't print anything useful. :facepalm:. Built it locally and used an fstring to get the value, it's 256.